pkg/option/config.go

```diff
@@ -1400,6 +1403,9 @@ type DaemonConfig struct {
 	// EnableWireguard enables Wireguard encryption
 	EnableWireguard bool
 
+	// WireguardRequireKernelMode enables the fallback to the userspace implementation
```

Comment:
Nit: this says "enables the fallback..." while the usage message in `daemon/cmd/daemon_main.go` says "disables the fallback...". I guess this should says "disables" as well:

```suggestion
	// WireguardRequireKernelMode disables the fallback to the userspace implementation
```

Reply:
Yes! Good catch!